common/app/components/App.tsx

```diff
@@ -1127,6 +1127,10 @@ function App({ origin, logoUrl, settings, extension, fetcher, onSettingsChanged
         );
     }, [keyBinder, handleOpenCopyHistory, ankiDialogOpen]);
 
+    useEffect(() => {
+        document.title = settings.tabName;
+    }, [settings]);
```

Comment:
Use `settings.tabName` instead of `settings` in the dependency array, so that the effect will run less often.

Reply:
Done.